docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -1,45 +1,33 @@
 # Adjusting quality settings
 
-The **Quality settings** give you control over the status of analyzed commits and pull requests, as well as the quality level thresholds in your repository dashboard. This feature also allows you to turn off metrics like duplication or complexity.
+The **Quality settings** allow you to configure quality gates for your pull requests and commits, and define quality goals that help you monitor the progress of the code quality in your repository dashboard.
 
-To access the quality settings, open your repository **Settings**, tab **Quality Settings**. On this page there are three sections where you can set specific values at the commit, pull request, and repository levels. Click **Edit settings** to edit the settings on each section.
+To access the quality settings, open your repository **Settings**, tab **Quality Settings**:
```

Comment:
@prcr suggestion: 

"To access the quality settings, open your repository **Settings**, and navigate to the tab **Quality Settings**:

Reply:
Here I'm deliberately and systematically using a [concise way of expressing navigation paths](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions#simple-instructions-with-right-angle-brackets) on the Codacy UI. Here are a couple of other examples:

- https://docs.codacy.com/repositories-configure/managing-branches/
- https://docs.codacy.com/repositories-configure/ignoring-files/
- https://docs.codacy.com/organizations/managing-people/

Since the navigation on the Codacy UI is generally straightforward and I also include screenshots highlighting the places where the user needs to click to reach the intended screen, I don't believe it's necessary to be verbose.